app/src/test/java/de/westnordost/streetcomplete/quests/AddPostboxCollectionTimesTest.kt

```diff
@@ -22,25 +24,37 @@ class AddPostboxCollectionTimesTest {
 
     @Test fun `apply collection times answer`() {
         questType.verifyAnswer(
-            CollectionTimes(OpeningHoursRuleList(listOf(
-                Rule().apply {
-                    days = listOf(WeekDayRange().also {
-                        it.startDay = WeekDay.MO
-                    })
-                    times = listOf(TimeSpan().also {
-                        it.start = 60
-                    })
-                },
-                Rule().apply {
-                    days = listOf(WeekDayRange().also {
-                        it.startDay = WeekDay.TU
-                    })
-                    times = listOf(TimeSpan().also {
-                        it.start = 60*2
-                    })
-                    isAdditive = true
-                },
-            ))),
+            CollectionTimes(
+                OpeningHoursRuleList(
+                    listOf(
+                        Rule().apply {
+                            days = listOf(
+                                WeekDayRange().also {
+                                    it.startDay = WeekDay.MO
+                                }
+                            )
+                            times = listOf(
+                                TimeSpan().also {
+                                    it.start = 60
+                                }
+                            )
+                        },
+                        Rule().apply {
+                            days = listOf(
+                                WeekDayRange().also {
+                                    it.startDay = WeekDay.TU
+                                }
+                            )
+                            times = listOf(
+                                TimeSpan().also {
+                                    it.start = 60 * 2
+                                }
+                            )
+                            isAdditive = true
+                        },
+                    )
+                )
+            ),
```

Comment:
This is another. TBH, I really don't like a linter that would force when to make a newline and when not to. This decision should be up to the human, because in the end, the human also needs to read this. Linters should really limit themselves to small stuff like removing dangling whitespaces, wrong indentation and stuff like this. Not stuff that may be a matter of taste.

Reply:
And again, the `indent` rule ([`9c2f170`](https://github.com/streetcomplete/StreetComplete/pull/3649/commits/9c2f170cfb7c138d01501e0c7ccbecc54f41b511#diff-906eaa22cdf62d42a511403ae9c99aa0e6cf1c69048042d59138f17ffb239e04)).